substrate/frame/collective/src/benchmarking.rs

```diff
@@ -77,7 +78,7 @@ benchmarks_instance_pallet! {
 					Box::new(proposal.clone()),
 					MAX_BYTES,
 				)?;
-				let hash = T::Hashing::hash_of(&proposal);
+				let hash = BlakeTwo256::hash_of(&proposal);
```

Comment:
Why not use T::Hashing anymore? The pallet preimage implementation uses T::Hash, not hardcoded to BlakeTwo256